docs/configuration/http_filters/cors_filter.rst

```diff
@@ -0,0 +1,61 @@
+.. _config_http_filters_cors:
+
+CORS filter
+====================
+
+This is a filter which handles Cross-Origin Resource Sharing requests based on route or virtual host settings.
+For the meaning of the headers please refer to the pages below.
+https://developer.mozilla.org/en-US/docs/Web/HTTP/Access_control_CORS
```

Comment:
nit: I would potentially use bulleted list here. (Either way take a look at generated docs locally to make sure they are to your liking).